lib/gui/app/components/safe-webview.js

```diff
@@ -191,12 +191,18 @@ class SafeWebview extends react.PureComponent {
    * @param {Event} event - Event object
    */
   didGetResponseDetails (event) {
-    const HTTP_OK = 200
-    const HTTP_ERR = 400
+    // This seems to pick up all requests related to the webview,
+    // only care about this event if it's a request for the main frame
```

Comment:
Does this trigger for all requests? Like also loading of HTML / CSS / images etc.?

Reply:
Yup, that's what I noticed now... I wonder if we should keep that for core requests (stylesheet, important scripts) to guarantee a good banner experience.

Reply:
@Shou Do you mean turn on the online banner if the HTML page loads okay, but then turn off the online banner if any of the CSS or image files fail to load? Maybe it's better to leave it always showing as soon as the HTML has loaded, to prevent any "flickering" if a user is on a slow connection and the last resource fails to load? \*shrug\*

Reply:
Although AFAICT all the resources on the HTML page are hosted on the same etcher.io domain anyway, so the chance of the HTML loading but any of the resources failing to load seems _very_ unlikely?

Reply:
I looked at all the requests and it seems we should be safe with assuming if the banner HTML loads, so will the other content. The things that might _not_ load are analytics related and as such not something that impacts users directly, so we should show the banner.